rule_packages/c/Concurrency4.json

```diff
@@ -0,0 +1,91 @@
+{
+  "CERT-C": {
+    "CON30-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "",
+          "kind": "problem",
+          "name": "Clean up thread-specific storage",
+          "precision": "medium",
+          "severity": "error",
+          "short_name": "CleanUpThreadSpecificStorage",
+          "tags": []
+        }
+      ],
+      "title": "Clean up thread-specific storage"
+    },
+    "CON34-C": {
+      "properties": {
+        "obligation": "rule" 
+      },
+      "queries": [
+        {
+          "description": "Accessing thread-local variables with automatic storage durations can lead to unpredictable program behavior.",
+          "kind": "problem",
+          "name": "Declare objects shared between threads with appropriate storage durations",
+          "precision": "high",
+          "severity": "error",
+          "short_name": "AppropriateThreadObjectStorageDurations",
+          "tags": [
+            "correctness",
+            "concurrency"
+          ],
+          "implementation_scope": {
+            "description": "This query does not consider Windows implementations or OpenMP implementations. This query is primarily about excluding cases wherein the storage duration of a variable is appropriate. As such, this query is not concerned if the appropriate synchronization mechanisms are used, such as sequencing calls to `thrd_join` and `free`. An audit query is supplied to handle some of those cases."
+          }
+        },
+        {
+          "description": "Storage durations not correctly initialized can cause unpredictable program behavior.",
+          "kind": "problem",
+          "name": "(Audit) Declare objects shared between threads with appropriate storage durations",
+          "precision": "high",
+          "severity": "error",
+          "short_name": "ThreadObjectStorageDurationsNotInitialized",
+          "tags": [
+            "external/autosar/audit",
```

Comment:
minor thing  - should this be "external/cert/audit" [instead](https://github.com/github/codeql-coding-standards/blob/main/schemas/rule-package.schema.json#L286)?

Reply:
Good catch!